receiver/postgresqlreceiver/testdata/scraper/otel/expected.json

```diff
@@ -0,0 +1 @@
e":"live"}}],"timeUnixNano":"1633639622621536000","asInt":"7"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table1"}},{"key":"state","value":{"stringValue":"dead"}}],"timeUnixNano":"1633639622621536000","asInt":"8"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"state","value":{"stringValue":"live"}}],"timeUnixNano":"1633639622621536000","asInt":"9"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"state","value":{"stringValue":"dead"}}],"timeUnixNano":"1633639622621536000","asInt":"10"}]}},{"name":"postgresql.operations","description":"The number of db row operations.","sum":{"dataPoints":[{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table1"}},{"key":"operation","value":{"stringValue":"ins"}}],"timeUnixNano":"1633639622621536000","asInt":"39"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table1"}},{"key":"operation","value":{"stringValue":"upd"}}],"timeUnixNano":"1633639622621536000","asInt":"40"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table1"}},{"key":"operation","value":{"stringValue":"del"}}],"timeUnixNano":"1633639622621536000","asInt":"41"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table1"}},{"key":"operation","value":{"stringValue":"hot_upd"}}],"timeUnixNano":"1633639622621536000","asInt":"42"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"operation","value":{"stringValue":"ins"}}],"timeUnixNano":"1633639622621536000","asInt":"43"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"operation","value":{"stringValue":"upd"}}],"timeUnixNano":"1633639622621536000","asInt":"44"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"operation","value":{"stringValue":"del"}}],"timeUnixNano":"1633639622621536000","asInt":"45"},{"attributes":[{"key":"database","value":{"stringValue":"otel"}},{"key":"table","value":{"stringValue":"public.table2"}},{"key":"operation","value":{"stringValue":"hot_upd"}}],"timeUnixNano":"1633639622621536000","asInt":"46"}],"aggregationTemporality":"AGGREGATION_TEMPORALITY_CUMULATIVE","isMonotonic":true}},{"name":"postgresql.rollbacks","description":"The number of rollbacks.","sum":{"dataPoints":[{"attributes":[{"key":"database","value":{"stringValue":"otel"}}],"timeUnixNano":"1633639622621536000","asInt":"2"}],"aggregationTemporality":"AGGREGATION_TEMPORALITY_CUMULATIVE","isMonotonic":true}}]}]}]}
```

Comment:
This is gonna be hard to manage or even to figure out what is the issue (if any) when test change, is there a way to generate these files?

Reply:
Currently, the way to regenerated is to use [scrapertest.WriteExpected](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e066c9cdd3ebc1fd288e1e74479cc3f6fa9caa61/internal/scrapertest/expected.go#L34). Perhaps we can find a better way, but there is some benefits to having a golden result like this.

@dehaansa Can you please regenerate the file? Note that `scrapertest.WriteExpected` will pretty print the json, which makes it much easier to read, and also will give us a line-by-line git history when expected changes occur.

Reply:
These files are generated using the receiver and the file exporter.